### PR TITLE
PlotPowerMonitors: add figsize, axis labels

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: src/Visualization/Python/PlotPowerMonitors.py [hunk 1]
```diff
@@ -299,6 +309,7 @@ def plot_power_monitors(
         "'--elements' / '-e' patterns."
     ),
 )
+@click.option("--figsize", nargs=2, type=float, help="Figure size in inches.")
```
Comment (knelli2) [inline src/Visualization/Python/PlotPowerMonitors.py]: Test the new option?

**Review by knelli2 — CHANGES_REQUESTED** [review]

File: src/Visualization/Python/PlotPowerMonitors.py (hunk 1 above)
Comment (nilsvu) [reply src/Visualization/Python/PlotPowerMonitors.py]: We don't currently have a way to test plot layouts

Comment (knelli2) [reply src/Visualization/Python/PlotPowerMonitors.py]: Sorry my bad, I wasn't clear. I meant test that passing in a value that isn't the default doesn't cause an error.

Comment (nilsvu) [reply src/Visualization/Python/PlotPowerMonitors.py]: ok sure

**Review by knelli2 — APPROVED** [review]
Looks good

**Pull request merged**